<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArraignmentContinueDialog</class>
 <widget class="QDialog" name="ArraignmentContinueDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2500</width>
    <height>3500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Arraignment Continuance Entry Case Information</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 61, 102);
font: 75 10pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(29, 61, 102);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>948</width>
           <height>685</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="case_name_Frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,2,1">
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Def. Last Name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="NoScrollDateEdit" name="plea_trial_date">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Def. Counsel:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="NoScrollComboBox" name="defense_counsel_type_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>Public Defender</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Private Counsel</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QCheckBox" name="defense_counsel_waived_checkBox">
               <property name="text">
                <string>Defendant appeared without counsel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="DefenseCounselComboBox" name="defense_counsel_name_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QPushButton" name="clear_fields_case_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Clear Fields</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="defendant_first_name_lineEdit">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="defendant_last_name_lineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Appearance Reason:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="cancel_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Case Number:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Def. First Name:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="case_number_lineEdit">
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="5">
              <widget class="Line" name="line">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="2">
              <widget class="NoScrollComboBox" name="appearance_reason_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>arraignment</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>final community control violation hearing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>preliminary community control violation hearing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>final pre-trial</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>LEAP sentencing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>jury trial</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>trial to court</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>motion hearing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>warrant for failure to appear</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="frame_8">
            <property name="styleSheet">
             <string notr="true">
background-color: rgb(255, 255, 127);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QPushButton" name="create_entry_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 255, 0);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Open Entry</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="close_dialog_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 96, 82);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Close Dialog</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;CONTINUANCE INFORMATION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="NoScrollComboBox" name="continuance_length_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>2 Weeks</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1 Week</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Continuance Length:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Continuance Reason:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="NoScrollComboBox" name="continuance_reason_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>allow defendant to obtain counsel</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>to arrange for an interpreter for defendant</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>New Arraignment Date:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="NoScrollDateEdit" name="new_arraignment_date_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoScrollDateEdit</class>
   <extends>QDateEdit</extends>
   <header>munientry.widgets.custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>NoScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>munientry.widgets.combo_boxes</header>
  </customwidget>
  <customwidget>
   <class>DefenseCounselComboBox</class>
   <extends>QComboBox</extends>
   <header>munientry.widgets.combo_boxes</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>defendant_first_name_lineEdit</tabstop>
  <tabstop>defendant_last_name_lineEdit</tabstop>
  <tabstop>case_number_lineEdit</tabstop>
  <tabstop>create_entry_Button</tabstop>
  <tabstop>close_dialog_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>create_entry()</slot>
  <slot>continue_dialog()</slot>
  <slot>close_window()</slot>
  <slot>set_dialog()</slot>
  <slot>update_case_information()</slot>
  <slot>ovi_dialog()</slot>
  <slot>sentencing_dialog()</slot>
  <slot>proceed_to_sentencing()</slot>
  <slot>proceed_to_ovi()</slot>
  <slot>add_charge()</slot>
  <slot>set_statute()</slot>
  <slot>set_pay_date()</slot>
  <slot>amend_offense()</slot>
  <slot>set_offense()</slot>
  <slot>add_charge_to_view()</slot>
  <slot>delete_charge()</slot>
  <slot>delete_charge_from_view()</slot>
  <slot>close_event()</slot>
  <slot>set_fra_in_file()</slot>
  <slot>set_fra_in_court()</slot>
  <slot>start_add_conditions_dialog()</slot>
  <slot>start_amend_offense_dialog()</slot>
 </slots>
</ui>
